src/style.css

```diff
@@ -2,0 +1,68 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: #3d4e61;
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.primary-search-bar {
+  height: 70px;
+}
+
+.secondary-search-bar {
+  height: 42px;
+}
+
+.search-bar-input {
+  width: 100%;
+  height: 100%;
+  background-repeat: no-repeat;
+  background-image: url(./images/Search.svg);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.search-bar-input--primary {
+  background-size: 19px 19px;
+  background-position: 26px center;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.search-bar-input--secondary {
+  background-size: 11px 11px;
+  background-position: 13px center;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.search-bar-input::placeholder {
+  color: #3d4e61;
+}
+
+.search-bar-input:hover {
+  box-shadow: rgba(0, 0, 0, 0.25);
+}
+
+.search-bar-input:focus {
+  box-shadow: rgba(0, 0, 0, 0.25);
+  font-family: Avenir-Heavy, sans-serif;
```

Comment:
You should provide a fallback font for 'Avenir-Heavy' in case it fails to load. This can be a system font like 'Arial', 'Helvetica', or 'sans-serif'. (CSS checklist)